content/en/docs/reference/access-authn-authz/kubelet-authn-authz.md

```diff
@@ -73,6 +73,25 @@ Kubelet API  | resource | subresource
 /spec/\*      | nodes    | spec
 *all others* | nodes    | proxy
 
+{{< feature-state feature_gate_name="KubeletFineGrainedAuthz" >}}
```

Comment:
I'd suggest we move these new contents into a H3 section `### Fine-grained authorization` and put the `feature-state` into that subsection.

This shortcode, in its current format, may become confusing. Readers cannot tell which paragraphs/sections are about the new feature.

Reply:
Done.